chart/values.yaml

```diff
@@ -69,46 +69,117 @@ controller:
+        kubernetes:
+          # Vault Auth Role to use
+          # This is a required field and must be setup in Vault prior to deploying the helm chart
+          # if `defaultAuthMethod.enabled=true`
+          # @type: string
+          role: ""
+
+          # Kubernetes ServiceAccount associated with the Transit Vault Auth Role
+          # Defaults to using the Operator's service-account.
+          # @type: string
+          serviceAccount:
+
+          # Token Audience should match the audience of the vault kubernetes auth role.
+          # @type: array<string>
+          tokenAudiences: []
+
+        # Vault JWT auth method specific configuration
+        jwt:
+          # Vault Auth Role to use
+          # This is a required field and must be setup in Vault prior to deploying the helm chart
+          # if using JWT for the Transit VaultAuthMethod.
+          # @type: string
+          role: ""
+
+          # One of the following is required prior to deploying the helm chart
+          # - K8s secret that contains the JWT
+          # - K8s service account if a service account JWT is used as a Vault JWT auth token and needs generating by VSO
+
+          # Name of Kubernetes Secret that has the Vault JWT auth token.
+          # The Kubernetes Secret must contain a key named `jwt` which references the JWT token, and must exist in the namespace
+          # of any consuming VaultSecret CR. This is a required field if a JWT token is provided.
+          # @type: string
+          secretName: ""
+
+          # Kubernetes ServiceAccount to generate a service account JWT
+          # @type: string
+          serviceAccount: "default"
+
+          # Token Audience should match the bound_audiences or the `aud` list in bound_claims if applicable
+          # of the Vault JWT auth role.
+          # @type: array<string>
+          tokenAudiences: [ ]
+
+        # AppRole auth method specific configuration
+        appRole:
+          # AppRole Role's RoleID to use for authenticating to Vault.
+          # This is a required field when using appRole and must be setup in Vault prior to deploying the
+          # helm chart.
+          # @type: string
+          roleId: ""
+          # Name of Kubernetes Secret that has the AppRole Role's SecretID used to authenticate with Vault.
+          # The Kubernetes Secret must contain a key named `id` which references the AppRole Role's
+          # SecretID, and must exist in the namespace of any consuming VaultSecret CR.
+          # This is a required field when using appRole and must be setup in Vault prior to deploying the
+          # helm chart.
+          # @type: string
+          secretName: ""
+
+        # Params to use when authenticating to Vault using this auth method.
+        # params: |
+        #   "vault-something1": "foo"
+        #   "vault-something2": "bar"
+        #   "vault-something3": "baz"
+        # @type: string
+        params: ""
```

Comment:
would it be possible to make these a map instead of a multiline string? Ideally we could update all others of this type.